packages/amplify_storage_plugin_interface/test/types/DownloadFile/downloadFileOptions_test.dart

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_storage_plugin_interface/src/DownloadFile/DownloadFileOptions.dart';
+import 'package:amplify_storage_plugin_interface/src/Storage/StorageAccessLevel.dart';
```

Comment:
This and other places, can we use a relative path here since it's in the same package?